developer-docs-site/docs/guides/running-a-local-multi-node-network.md

```diff
@@ -0,0 +1,125 @@
+---
+title: "Running a Local Mult-node Network"
+slug: "running-a-local-multi-node-network"
+---
+
+# Running a Local Mult-node Network
+
+This guide describes how to run a local network with multiple validator nodes and validator fullnodes. You will use the [Aptos Forge CLI](https://github.com/aptos-labs/aptos-core/tree/main/testsuite/forge-cli) for this.
+
+:::tip Use only for test deployments
+The method described in this guide should be used only for test deployments of multi-node local network. Do not use this guide for deploying in production environments. Currently this is the only guide for multi-node deployments. 
```

Comment:
nit: `deployments of multi-node local network.` -> `deployments of multi-node local networks.`